app/src/main/java/nerd/tuxmobil/fahrplan/congress/designsystem/colors/ColorSchemes.kt

```diff
@@ -16,12 +16,11 @@ internal fun darkColorScheme() = androidx.compose.material3.darkColorScheme(
     onSurface = colorResource(R.color.text_primary), // used by ListItem -> headlineContent
     onSurfaceVariant = colorResource(R.color.text_secondary), // used by SearchBarDefaults.InputField placeholder, ListItem -> overlineContent
     inverseOnSurface = colorResource(R.color.session_item_text_on_highlight_background), // used by SessionCard
-    outline = colorResource(R.color.colorAccent), // used by SearchBarDefaults.InputField divider
```

Comment:
Can you please state in the commit message why we don't want the color to be overwritten?

Reply:
Done.